cmd/podman/containers/run.go

```diff
@@ -106,6 +107,10 @@ func run(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
+	if cliVals.TTY && cliVals.Interactive && !terminal.IsTerminal(int(os.Stdin.Fd())) {
+		return errors.New("the input device is not a TTY")
+	}
+
```

Comment:
I am fine with this check, it is the same `tty` does, but it won't help in the case where the output is sent to a pipe like:

`podman run -ti ... | another-command`

Should we have another check for stdout?

Reply:
Docker does not do anything in this case.